src/lib.rs

```diff
@@ -415,6 +415,46 @@ macro_rules! __bitflags {
     };
 }
 
+#[macro_export(local_inner_macros)]
+#[doc(hidden)]
+#[cfg(const_fn)]
+macro_rules! __fn_bitflags {
+    (
+        $(#[$filtered:meta])*
+        fn $($item:tt)*
```

Comment:
At some point I think we'll find ourselves bumping our minimum supported rustc version past `1.31.0` and won't need this anymore. So it might make sense just to require the `const` keyword in the input tokens for both `__fn_bitflags` macro impls even though older compilers won't be generating `const` functions:

```rust
__fn_bitflags! {
    #[inline]
    pub const fn bits(&self) -> $T {
        self.bits
    }
}
```